test/cpp/end2end/end2end_test.cc

```diff
@@ -338,7 +338,11 @@ class End2endTest : public ::testing::TestWithParam<TestScenario> {
         kMaxMessageSize_);  // For testing max message size.
   }
 
-  void ResetChannel() {
+  void ResetChannel(
+      std::vector<
+          std::unique_ptr<experimental::ClientInterceptorFactoryInterface>>
+          interceptor_creators = std::vector<std::unique_ptr<
```

Comment:
I think you can just do `= {}` to set the default argument to be the empty vector. It works on some simple godbolt codes.